test/manifests/setup.yaml

```diff
@@ -60,6 +61,13 @@ spec:
     image: public.ecr.aws/karpenter/tools:latest
     script: |
       cmd="create"
+      
+      # Use a bigger instance type when running scale tests so we can have more cpu/memory
+      instanceType="c5.large"
+      if [[ "$(params.test-filter)" == "Scale" ]]; then
+        instanceType="c5.2xlarge"
+      fi
```

Comment:
I feel like it's fine just to remove this and just use a `.2xlarge` for all tests, no need to complicate the logic

Reply:
Sounds good. This is a short-term change anyways since removing Tekton is probably down-the-pipe.